### ISSUE TITLE
Request: FontAwesome 5 & FontAwesome Pro

### ISSUE THREAD

**Issue by ShakataGaNai** [ryanoasis/nerd-fonts#221]
#### 🎯 Subject of the issue

I'd like to put in a request for support of FontAwesome 5 and FontAwesome Pro. The later requires users licenses, so perhaps just support in the patcher.

**Comment by ryanoasis** [comment]
@ShakataGaNai Yes this is a very big idea! :+1:  I am aware of them but haven't done any movement on that front. :sweat_smile:

**Comment by ryanoasis** [comment]
I think we can probably rule out FA Pro (I don't feel confident exploring that route -- unless someone can convince me otherwise).

As for FA 5 perhaps we can give an option with the patcher, something like either FA 5 or the Fork Awesome (this was brought up in #241). FA 4.7 fork that is arguably more open.

I think we need to discuss pros/cons of each. I, myself, would need to read up on the direction of each project. Perhaps we need to choose one over the other, or at least one as the default. 

Usually we try not to care about a few duplicates between sets here and there.. however here I think it would be overkill to support both out of the box (in the released patched fonts).

**Comment by maximbaz** [comment]
> Usually we try not to care about a few duplicates between sets here and there.. however here I think it would be overkill to support both out of the box (in the released patched fonts).

Given that FA 5 has new design for many icons, maybe it's not that bad idea to have both FA5 and ForkAwesome (based on FA 4) in the released fonts — there will be two icons for everything, but they will have different design after all. I've heard people who like the old design more.

Still, I'd like to see FA 5 be the "default" font, and let its icons stay in the codepoints currently allocated for FA. ForkAwesome should be put in a new location within Nerd Fonts.

**Comment by bvgastel** [comment]
Any news about this? I would really like the new icons in v5.

**Comment by aiguofer** [comment]
I'd love to use https://fontawesome.com/icons/memory?style=solid; after a while of digging around I found the reason that symbol is not available is because it was released in FA 5. Would be really interested in this!

**Comment by brennanfee** [comment]
Given the open-source focus of Nerd Fonts (especially from a redistribution perspective), my vote would be a switch away from FontAwesome and toward ForkAwesome.  If for no other reason than at least we would be on slid ground from a licensing and future access level.  Who knows if\when FontAwesome will make a\another profit driven decision to limit further the images they provide or the manner in which they can be used (non-commercial only is a common tactic).

Given that ForkAwesome was created expressly to be free\open and intends to stay that way it would make more sense to shift over to using that instead.

**Comment by kowalski7cc** [comment]
I think FontAwesome has way more icons (event in the free version) than ForkAwesome (1500+ vs 700+).
And I don't find FontAwesome license so restrictive (CC BY 4.0)

**Comment by TechieAndroid** [comment]
I've tried asking on here and haven't gotten any answer yet but I was pointed here to ask.

Is it because font awesome hasn't been updated in nerd fonts that none of the glyphs work?
https://fontawesome.com/cheatsheet

**Comment by Soooda** [comment]
Just spent the whole night and found out FA 5 has not been included yet xD
I would definitely vote for updating to FA5. PLEASE!

**Comment by Erothas** [comment]
After two days of trying to figure out why my status bar icons break whenever I install nerd fonts, I finally figured out that it's because the damn awesome icons I use are not updated in nerd fonts. It has been over THREE YEARS since this issue was started. At this point, the owner will just not do it for whatever reason that I could not care less about. And I would honestly be more angry if I give up now after wasting two whole days for no reason.

So, Is there any way to do this manually? If anyone could please direct me towards the right way so we can do it ourselves. That would be great.

**Comment by paulhersch** [comment]
Same as Erothas, commenting for notification

**Comment by ShakataGaNai** [comment]
FYI - On the right hand side of this page (and any issue on GitHub) there is a section labeled "Notification" with a big green "Subscribe" button. That way you can tune into an issue without spamming.

**Comment by ryanoasis** [comment]
converting this from an issue to a discussion
